style/sass/scss-lint-config.yml

```diff
@@ -0,0 +1,237 @@
+# Up-to-date with SCSS-Lint v0.43.2
```

Comment:
How likely is it that that this version will grow stale? Is the idea here to bump it when Hound supports newer versions?